packages/nodes-base/nodes/HttpRequest/HttpRequest.node.ts

```diff
@@ -16,12 +16,13 @@ export class HttpRequest extends NodeVersionedType {
 			group: ['input'],
 			subtitle: '={{$parameter["requestMethod"] + ": " + $parameter["url"]}}',
 			description: 'Makes a HTTP request and returns the received data',
+			defaultVersion: 2,
```

Comment:
having a defaultVersion makes sense. would this enable rollbacks? or canary testing before a version becomes `default`?